src/index.html

```diff
@@ -15,23 +15,21 @@ <h1 class="headingText">Leaderboard</h1>
       <section class="scoreBoardContainer">
         <div class="recentScore">
           <h2 class="recentScoreText">Recent scores</h2>
-          <button class="recentScoreBtn">Refresh</button>
+          <button class="refreshBtn">Refresh</button>
         </div>
         <article class="scoreBoard"></article>
       </section>
       <section class="scoreInputContainer">
         <div class="scoreInputText">
           <h2>Add your score</h2>
         </div>
-        <div class="scoreInput">
+        <form class="scoreInput">
           <input type="text" placeholder="Your name" class="inputName" />
           <input type="text" placeholder="Score" class="inputScore" />
```

Comment:
- [Optional] Kindly, Validate the Name and score fields to ensure a user cannot submit blank entries and the wrong value for the score as the [API](https://microverse.notion.site/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) accepts number only. 👍

Reply:
Ok, sir thanks a lot I will do it.